internal/pkg/cli/pipeline_init.go

```diff
@@ -245,151 +222,31 @@ func (o *initPipelineOpts) RecommendedActions() []string {
 	}
 }
 
-func (o *initPipelineOpts) createSecretName() string {
-	return fmt.Sprintf("github-token-%s-%s", o.appName, o.githubRepo)
-}
-
-func (o *initPipelineOpts) createPipelineName() string {
-	return fmt.Sprintf("pipeline-%s-%s-%s", o.appName, o.githubOwner, o.githubRepo)
-}
-
-func (o *initPipelineOpts) createPipelineProvider() (manifest.Provider, error) {
-	config := &manifest.GitHubProperties{
-		OwnerAndRepository:    "https://" + githubURL + "/" + o.githubOwner + "/" + o.githubRepo,
-		Branch:                o.gitBranch,
-		GithubSecretIdKeyName: o.secretName,
-	}
-	return manifest.NewProvider(config)
-}
-
-func (o *initPipelineOpts) getEnvConfig(environmentName string) (*config.Environment, error) {
-	for _, env := range o.envs {
-		if env.Name == environmentName {
-			return env, nil
-		}
-	}
-	return nil, fmt.Errorf("environment %s in application %s is not found", environmentName, o.appName)
-}
-
-func (o *initPipelineOpts) createPipelineManifest() error {
-	pipelineName := o.createPipelineName()
-	provider, err := o.createPipelineProvider()
-	if err != nil {
-		return fmt.Errorf("create pipeline provider: %w", err)
-	}
-
-	var stages []manifest.PipelineStage
-	for _, environmentName := range o.environments {
-		env, err := o.getEnvConfig(environmentName)
+func (o *initPipelineOpts) askEnvs() error {
+	if len(o.environments) == 0 {
```

Comment:
```suggestion
	if len(o.environments) != 0 {
	    return nil
	}
	...
```
Benefit is we can indent less lines of code.

Reply:
Done.